site/it/tutorials/customization/basics.ipynb

```diff
@@ -1,480 +1,480 @@
+        "Se avete suggerimenti per migliorare questa traduzione, mandate per favore una pull request al repository Github [tensorflow/docs](https://github.com/tensorflow/docs). \n",
+        "Per proporsi come volontari alla scrittura o alla review delle traduzioni della comunità contattate la \n",
+        "[mailing list docs@tensorflow.org](https://groups.google.com/a/tensorflow.org/forum/#!forum/docs)."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "6sILUVbHoSgH"
+      },
+      "source": [
+        "Questo è un tutorial introduttivo su TensorFlow che mostra come:\n",
+        "\n",
+        "* Importare i pacchetti richiesti\n",
+        "* Creare e usare tensori\n",
+        "* Usare l'accelerazione GPU\n",
+        "* Dare una dimostrazione del `tf.data.Dataset`"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "miTaGiqV9RjO"
+      },
+      "outputs": [],
+      "source": [
+        "from __future__ import absolute_import, division, print_function\n",
+        "\n",
+        "try:\n",
+        "  # %tensorflow_version only exists in Colab.\n",
+        "  %tensorflow_version 2.x\n",
+        "except Exception:\n",
+        "  pass\n"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "z1JcS5iBXMRO"
+      },
+      "source": [
+        "## Importare TensorFlow\n",
+        "\n",
+        "Per iniziare, importa il modulo `tensorflow`. A partire da TensorFlow 2.0, l'esecuzione eager è attiva di default. Questo permette un frontend più interattivo verso TensorFlow, sui quali dettagli si discuterà oltre."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 0,
+      "metadata": {
+        "colab": {},
+        "colab_type": "code",
+        "id": "vjBPmYjLdFmk"
+      },
+      "outputs": [],
+      "source": [
+        "import tensorflow as tf"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "H9UySOPLXdaw"
+      },
+      "source": [
+        "## Tensori\n",
+        "\n",
+        "Un Tensore è un array multi-dimensionale. Similmente all'oggetto `ndarray` di NumPy, gli oggetti `tf.Tensor` hanno un tipo di dato e una forma. Inoltre, gli oggetti `tf.Tensor` possono risiedere in un acceleratore di memoria (come una GPU). TensorFlow offre una libreria ricca di operazioni ([tf.add](https://www.tensorflow.org/api_docs/python/tf/add), [tf.matmul](https://www.tensorflow.org/api_docs/python/tf/matmul), [tf.linalg.inv](https://www.tensorflow.org/api_docs/python/tf/linalg/inv) etc.) che consumano e producono oggetti `tf.Tensor`. Queste operazioni convertono automaticamente tipi nativi di Python, ad esempio:\n"
```

Comment:
Similmente all'oggetto `ndarray` di NumPy, gli oggetti `tf.Tensor` hanno un tipo di dato e una forma --> Come negli oggetti `ndarray` di NumPy, gli oggetti `tf.Tensor` hanno un tipo di dato e un formato

Reply:
Yes, the proposed sentence sounds better, thanks :+1: